charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,100 @@
+
+*Data* - raw data files from an experiment such as fastq files for sequencing experiments or tiff stacks for imaging experiments
+
+*Metadata* - descriptive information about what projects, samples and experiments are and how they were collected or conducted
+
+*New data types* - experimental data types which either can't already be ingested by the DCP or have only recently (~6m - 1yr) started to be ingested by the system
+
+*Established data types* - experimental data types which have a history of submission and downstream use in the DCP and there are already many datasets of this type represented in the platform
+
+## Objectives
+
+- Supporting data contributors uploading data and metadata to the DCP
+- Helping data contributors fix errors with their data and metadata before it can be submitted to the DCP
+- Ensuring a data contributor's experiment can be accurately represented in the DCP
+- Working with the data contributor to propose extensions to the HCA metadata schema if additional metadata fields are required
+- Working with the metadata team to review proposed metadata extensions to ensure they meet the needs of data contributors
+- Working with the UX team to collect and provide feedback from data contributors to the ingest infrastructure team about how the infrastructure can be improved
+- Working with the UX team to ensure the ingest process provides a positive user experience to the data contributors
+
+## In-scope
+
+### For new data types
+
+- Working with data contributors to define a spreadsheet template to collect their metadata
+- Collecting metadata from data contributors in HCA ingest spreadsheet form
+- Collecting data files from data contributors
+- Reviewing collected data files and metadata
+- Submitting data files and metadata to the DCP on behalf of a data contributor
+- Working with data contributors to fix errors in the data and metadata
+
+### For established data types
+
+- Supporting data contributors uploading data files to the platform
+- Supporting data contributors using the Ingest User interface to generate a metadata template spreadsheet
+- Supporting data contributors uploading metadata to the platform
+- Supporting data contributors who wish to use the Ingest API to submit data
+- Working with data contributors to fix errors in their data and metadata
+
+### User Experience
+
+- Collecting feedback about the submission process from data contributors
+- Working with the UX team to use feedback to define new requirements for the system to be past to the PM team
+
+### Documentation and Training
+
+- Writing training material about how to submit data to the DCP
+- Responding to data contribution requests on Zendesk
```

Comment:
Why is _Responding to data ... requests_ under **Documentation and Training**?

Reply:
I was trying to avoid duplication but then I broke that rule later. I have now merged this with another group of duplicates under all data contributors